spec/Section 6 -- Execution.md

```diff
@@ -656,6 +656,12 @@ As an example, this might accept the {objectType} `Person`, the {field}
 {"soulMate"}, and the {objectValue} representing John Lennon. It would be
 expected to yield the value representing Yoko Ono.
 
+List values are resolved similarly. For example, {ResolveFieldValue} might also
```

Comment:
I do not like "similarly" because technically it follows the exact same algorithm, whereas "similarly" implies that there _is_ a difference. Let's try alternate wording:

```suggestion
For an example relating to list fields, {ResolveFieldValue} might
```

Though I'm not convinced we need this paragraph.

Reply:
I just removed the whole paragraph.